tuf/keys/db.go

```diff
@@ -58,6 +58,15 @@ func (db *KeyDB) AddRole(r *data.Role) error {
 	return nil
 }
 
+// GetAllRoles gets all roles from the database
+func (db *KeyDB) GetAllRoles() []*data.Role {
+	roles := []*data.Role{}
```

Comment:
Non-blocking, as this is totally fine - I think you can also use `copy()` here - it probably does the same thing :D